rbac/management/migrations/0020_add_unique_to_display_name.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.4 on 2020-08-04 14:42
+
+from django.db import migrations, models
+
+
+def update_display_name(apps, schema_editor):
+    # get role model
+    Role = apps.get_model("management", "Role")
+
+    # iterate through all Access
+    for role in Role.objects.all():
+        # get the permission context from the permission
+        role.display_name = role.name
+        role.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("management", "0019_recreate_display_name_field"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Do we need both `AlterField` calls here or can the default/max/unique be set in one?